core-bundle/src/Resources/contao/elements/ContentModule.php

```diff
@@ -98,9 +100,9 @@ public function generate()
 
 		$strBuffer = $objModule->generate();
 
-		if (isset($objStopwatch))
+		if (isset($objStopwatch) && $objStopwatch->isStarted($strStopWatchId))
```

Comment:
Did you find out why the stopwatch isn't started at this point? Is something during `$objModule->generate();` stopping it?

Reply:
No, but as I understand it: different requests do have different stopwatch sections and there we get the little mess: https://github.com/symfony/symfony/issues/36623

Reply:
In my real life demo all stopwatches get started, but after the first subrequest all following `$stopwatch->stop()` will fail.

Reply:
I can follow this: https://github.com/symfony/symfony/issues/36762#issuecomment-626352054

Reply:
Ah yes, missed the linked ticket. Thanks!